package-lock.json

```diff
@@ -3074,7 +3073,7 @@
     },
     "node_modules/@mattermost/react-native-network-client": {
       "version": "0.1.0",
-      "resolved": "git+ssh://git@github.com/mattermost/react-native-network-client.git#aaf88d2142e8b5c3b89b17aa19605232c490b763",
+      "resolved": "git+ssh://git@github.com/mattermost/react-native-network-client.git#c17f1c042ac8c29a16e9222d4c1b172e93987b64",
```

Comment:
this reference fixes the crash seen by @jgilliam17 

the actual fix can be seen here https://github.com/mattermost/react-native-network-client/commit/c17f1c042ac8c29a16e9222d4c1b172e93987b64